dbus-wait.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+wait_for_dbus() {
```

Comment:
I can't think of a clean way to remove code duplication between this and [hm-diag](https://github.com/NebraLtd/hm-diag/pull/149/files#diff-a84b061bc6245cd09854d50b5257a6e60418bbdc76ec91788faae259588be97dR5) without a lot of overtooling. Eventually creating a APT release similar to hm-pyhelper is one long term thought but def overkill now.

Reply:
It's also in hm-miner as well just FYI but yeah couldn't think of an easy way to remove duplicate

Reply:
Having said that, the one in hm-miner checks the system bus instead of the session bus